src/sentry/static/sentry/app/views/onboarding/components/firstEventIndicator.tsx

```diff
@@ -7,43 +7,65 @@ import {IconCheckmark} from 'app/icons';
 import {t} from 'app/locale';
 import pulsingIndicatorStyles from 'app/styles/pulsingIndicator';
 import space from 'app/styles/space';
-import {Group, Organization} from 'app/types';
+import {Group} from 'app/types';
 import EventWaiter from 'app/utils/eventWaiter';
 import testableTransition from 'app/utils/testableTransition';
 
 type EventWaiterProps = Omit<React.ComponentProps<typeof EventWaiter>, 'children'>;
 type FirstIssue = null | true | Group;
 
-const FirstEventIndicator = (props: EventWaiterProps) => (
+type RenderProps = {
+  indicator: React.ReactNode;
+  firstEventButton: React.ReactNode;
+};
+
+type Props = EventWaiterProps & {
+  children: (props: RenderProps) => React.ReactNode;
```

Comment:
This has become a render-prop style component so we have more control over the layout. at the parent level